code/modules/clothing/head/winter_hats.dm

```diff
@@ -10,14 +10,14 @@
 	flags_inv = HIDEEARS
 	var/earsup = 0
 
-/obj/item/clothing/head/ushanka/nyakas
-	name = "visegradi nyakas"
+/obj/item/clothing/head/ushanka/cap
+	name = "ear-flap cap"
```

Comment:
if it's a visegradi hat you should keep visegradi in the name, otherwise people think it's generic

Reply:
i'll put it back in for you, but personally i don't like putting visegradi in front because it's implied to be origin restricted where it's actually mostly generic but linked to a culture